client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1978,7 +1982,13 @@
                 });
 
                 metadata.initialize().then(function() {
+                    scope.item.hasCropes = false;
                     scope.metadata = metadata.values;
+                    _.each(scope.metadata.crop_sizes, function(crop) {
+                        if (scope.item.renditions[crop.name]){
+                            scope.item.hasCropes = true;
+                        }
+                    })
```

Comment:
You can just use the built-in [some()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) method here, more concise:
```js
scope.metadata = metadata.values;
scope.item.hasCrops = scope.metadata.crop_sizes.some(function (crop) {
    return scope.item.renditions[crop.name];
});
```